shiksha-migration/update-intersted-to-join.js

```
@@ -0,0 +1,102 @@
+        if (sourceResult.rows.length === 0) {
+            console.log('[UPDATE] Nothing to update. Exiting.');
+            return;
+        }
+
+        let updated = 0;
+        let skipped = 0;
+
+        for (const row of sourceResult.rows) {
+            const userId = row.userId;
+
+            // Extract the scalar value from the array/text stored in FieldValues
```

ESLint

[error] Replace `············` with `······`

(prettier/prettier)